cloudferry_devlab/cloudferry_devlab/tests/base.py

```diff
@@ -255,12 +255,20 @@ def get_volume_snapshot_id(self, snapshot_name):
         raise test_exceptions.NotFound('Snapshot with name "%s" was not found'
                                        % snapshot_name)
 
-    def get_user_tenant_roles(self, user):
+    def get_user_tenant_roles(self, user, tenant_attrib=None):
```

Comment:
Nope. According to implementation those clearly should be two independent methods.